.github/workflows/ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Run tests
+ 
+on: [push]
```

Comment:
Can you also include the workflow to run when a pull request is created?  That way the user creating the pull request can immediately see the test run for their PR, and ultimately see if the PR passes or not

Reply:
Haven't tested this: https://stackoverflow.com/questions/70964103/trigger-github-workflow-on-pull-request

Reply:
done